snakemake/workflow.py

```diff
@@ -1296,7 +1317,11 @@ def execute(
             if success:
                 if self.dryrun:
                     if len(self.dag):
-                        logger.run_info("\n".join(self.dag.stats()))
+                        stats_msg, stats_dict = self.dag.stats()
+                        logger.warning(
```

Comment:
logger.info here, correct?

Reply:
Warning was correct to coerce it yellow, but coloring based on the log type will resolve this. I will push some changes soon to address this.

Reply:
resolved w/ LogEvent enum